test/instrumentation/modules/aws-sdk/s3.test.js

```diff
@@ -228,7 +228,7 @@ tape.test('simple S3 usage scenario', function (t) {
                   resource: 's3/elasticapmtest-bucket-1',
                 },
               },
-              http: { status_code: 200, response: { encoded_body_size: 58 } },
+              http: { status_code: 200 },
```

Comment:
Interesting. I vaguely recall seeing changes like this before. I don't recall why the change.